Convex_hull_3/include/CGAL/convex_hull_3.h

```diff
@@ -848,7 +848,7 @@ ch_quickhull_face_graph(std::list<typename Traits::Point_3>& points,
       CGAL_assertion( traits.has_on_positive_side_3_object()(
             construct_plane(v2->point(),v1->point(),v0->point()),
             v3->point()) );
-      make_tetrahedron(v0->point(),v1->point(),v3->point(),v2->point(),P);
+      // AF make_tetrahedron(v0->point(),v1->point(),v3->point(),v2->point(),P);
```

Comment:
When I uncomnent the example no longer compiles. @sloriot can you please have a look.

Reply:
I guess that if the class was in another namespace than CGAL it would work.  Maybe some `enable_if` magic  can help?

Reply:
Issue was that `halfedge_descriptor` must exist so as the generic version of `make_tetrahedron()` must be instantiated, even if it is eliminated later to pick the new one defined.